AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm/README.md

```diff
@@ -26,8 +26,6 @@ In this sample, you will create and run the general genetic algorithm and optimi
 
 This sample code is implemented for GPUs using Python. The sample assumes you have numba-dpex installed inside a Conda environment, similar to what is installed with the Intel® Distribution for Python*.
 
->**Note**: Read *[Get Started with the Intel® Distribution for Python*](https://www.intel.com/content/www/us/en/developer/articles/technical/get-started-with-intel-distribution-for-python.html)* to find out how you can achieve performance gains through Intel optimizations.
-
 The sample tutorial contains one Jupyter Notebook and one Python script. You can use either.
 
 ## Environment Setup
```

Comment:
step 2 line 43 - probably it should be 'intelpython3' instead of 'intelpython'

Reply:
step 3 - this step looks redundant because we only have 1 env for IDP and user is activating it on step 2